tests/Fixer/FunctionNotation/RegularCallableCallFixerTest.php

```diff
@@ -209,6 +209,26 @@ function call_user_func($foo){}
             '<?php foo(1,);',
             '<?php call_user_func("foo", 1,);',
         ];
+
+        yield 'empty string double quote' => [
+            '<?php call_user_func("", 1,);',
+        ];
+
+        yield 'empty string single quote' => [
+            '<?php call_user_func(\'    \', 1,);',
+        ];
+
+        yield 'string with padding' => [
+            '<?php call_user_func(" padded  ", 1,);',
+        ];
+
+        yield 'binary string lower double quote' => [
+            '<?php call_user_func(b"foo", 1,);',
```

Comment:
Shouldn't this one be fixed to `<?php foo(1,);`?

Reply:
It could, sure, however we have a dedicated fixer for removing `b"`. I like the more single-responsibility setup for a rule.

Also this case is already covered now (using two rules with priority setup to ensure proper working).

My hope is also that the whole binary string will be removed from PHP, when that is done than we only need to remove the one dedicated rule for it and we won't have to change all the rules that are handling it by themselves as well.

I raised the PR because some of the rules break valid PHP code, stopping these rules from doing that is more important to me than extending the cases-supported by these rules.